client/tests/phpunit/Pact/SiriusApiGatewayClientTest.php

```diff
@@ -284,8 +307,14 @@ private function setUpChecklistPdfPactBuilder(string $caseRef, string $reportPdf
             ]);
 
         $this->builder
-            ->uponReceiving('A submitted checklist pdf')
+            ->uponReceiving('A submitted checklist pdf  a')
```

Comment:
Is this a typo?

Reply:
Ah yeah - I was forcing a new check during dev as I'd already had a successful test run so pact wouldn't run a further test with the same name. I'll revert.